assets/templates/partials/previousreleases/contents/navigation.tmpl

```diff
@@ -0,0 +1,14 @@
+<aside class="ons-toc-container" role="complementary">
+  <a class="ons-skip-link" href="#summary">Skip to guide content</a>
```

Comment:
No need for a skip link here. It has been removed from the example on the [design system](https://ons-design-system.netlify.app/components/table-of-contents/examples/table-of-contents-sticky/)